compose/backend.yml

```diff
@@ -120,7 +146,6 @@ services:
     environment:
       DB_USERNAME:
       DB_PASSWORD:
-      POSTGRES_HOST_AUTH_METHOD: trust
```

Comment:
why remove?

Reply:
I think, it was removed by mistake I have reverted back.